pkg/options/options.go

```diff
@@ -43,8 +53,8 @@ type APIKeys struct {
 }
 
 type AWSAPIKey struct {
-	AWSAccessID     string `yaml:"awsAccessKeyID"`
-	AWSAccessSecret string `yaml:"awsSecretAccessKeyID"`
+	AWSAccessKeyID  string `yaml:"aws_access_key_id"`
```

Comment:
Wondering why you are changing these 2 yaml keys?  canary test and others are currently using the existing keys to build their options.yaml file so if you make this change for e2e it will require canary changes and changes for anyone else using the existing key names.

Reply:
ok